packages/minimongo/minimongo.js

```diff
@@ -97,6 +97,10 @@ LocalCollection.Cursor = function (collection, selector, options) {
     // stash for fast path
     self._selectorId = selector;
     self.matcher = new Minimongo.Matcher(selector);
+  } else if (LocalCollection._selectorIsIdPerhapsAsObject(selector)) {
+    // also do the fast path for { _id: idString }
+    self._selectorId = selector._id;
+    self.matcher = new Minimongo.Matcher(selector);
```

Comment:
might as well pull this out of the conditional (put it before the conditional), now it's three times!